sources/packages/backend/libs/integrations/src/esdc-integration/e-cert-integration/e-cert-part-time-integration/e-cert-files/e-cert-file-record.ts

```diff
@@ -122,27 +122,27 @@ export class ECertPartTimeFileRecord extends ECertFileRecord {
   /**
    * Sum of all CSGP grants + BC Provincial Part-time grants from this certificate.
    */
-  totalGrantAmount: number;
+  totalCanadaGrantAmount: number;
```

Comment:
This one states that represents Canada + Provincial grants, is there a reason why it was changed to indicate "Canada" only?

Reply:
I changed to `totalCanadaAndProvincialGrantsAmount`. Thanks.